### PR TITLE
Update for newest quip version

### PR DESCRIPTION
**Pull request by t-young31**
The latest version of QUIP/gap_fit/quippy should probably be the default

### CODE REVIEW

File: gaptrain/gap.py [hunk 1]
```diff
@@ -47,7 +52,7 @@ def ase_gap_potential_str(self):
         if not os.path.exists(f'{self.name}.xml'):
             raise IOError(f'GAP parameter file ({self.name}.xml) did not exist')
 
-        return ('pot = quippy.Potential("IP GAP", \n'
+        return (f'pot = {potential_class}("IP GAP", \n'
                 f'              param_filename="{self.name}.xml")')
```
Comment (tristan-j-wood) [inline gaptrain/gap.py:56]: The indentation seems a bit extreme here? Is it aligned with the brackets, I can't tell?

File: gaptrain/gap.py [hunk 2]
```diff
@@ -545,14 +559,14 @@ def ase_gap_potential_str(self):
 
         pt_str = ''
         for i in range(2):
-            pt_str += (f'pot{i+1} = quippy.Potential("IP GAP", \n'
+            pt_str += (f'pot{i+1} = {potential_class}("IP GAP", \n'
                        f'          param_filename="{self[i].name}.xml")\n')
```
Comment (tristan-j-wood) [inline gaptrain/gap.py:563]: Same indentation question as above

File: gaptrain/gap.py [hunk 3]
```diff
@@ -607,9 +621,9 @@ def ase_gap_potential_str(self,
         here = os.path.abspath(os.path.dirname(__file__))
         pt = open(os.path.join(here, 'iicalculator.py'), 'r').readlines()
 
-        pt += [f'inter_gap = quippy.Potential("IP GAP", '
+        pt += [f'inter_gap = {potential_class}("IP GAP", '
```
Comment (tristan-j-wood) [inline gaptrain/gap.py:624]: Probably fine, but some string lines have \n and some don't

File: gaptrain/gap.py (hunk 1 above)
Comment (t-young31) [reply gaptrain/gap.py:56]: it's not quite right (and depends on the `potential_class`) but it's useful to have some indentation for readability. The file is only ever written to a temporary directory anyway

File: gaptrain/gap.py (hunk 3 above)
Comment (t-young31) [reply gaptrain/gap.py:624]: mehh

**Pull request merged**